src/perlmodgen.cpp

```diff
@@ -1835,6 +1835,8 @@ void PerlModGenerator::generatePerlModForClass(const ClassDef *cd)
 
   m_output.openHash()
     .addFieldQuotedString("name", cd->name());
+  /* DGA: fix # #7547 Perlmod does not generate "kind" information to discriminate struct/union */
```

Comment:
please do not add comments like this in the code

Reply:
Ok, you mean to not add my shortname ("DGA") ?
But it is a good practice to still mention the fixed issue ?